app/helpers/likes_helper.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+module LikesHelper
+  def current_user_like?(object)
```

Comment:
- [ ] There should be no application logic in helpers. -- 
***Helpers are the place for view-related logic  not application-related logic. So it's better to move this method into `user` model because model can interact with databse directly.***

Reply:
Thanks for your tip. Unfortunatelly we couldn't make it work from the models so we will keep it in the helpers for now.